lib/data/models/mybudget.js

```diff
@@ -4,10 +4,11 @@ import MyBudgetItem from './mybudget-item'
 
 const log = debug('presupuesto:data:models:mybudget')
 let Schema = mongoose.Schema;
-let ObjectId = Schema.ObjectId;
+//let ObjectId = Schema.ObjectId;
 
 let MyBudgetSchema = new Schema({
-	rows: { type: [MyBudgetItem] },
+	rows: { type: [Schema.Types.Mixed] },
```

Comment:
For some reason it was throwing a CastException in my case when using [MyBudgetItem].